pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -40,9 +40,46 @@ func Validate(ic *types.InstallConfig) error {
 			field.Invalid(ovirtPlatformPath.Child("vnicProfileID"), ic.Ovirt.VNICProfileID, err.Error()))
 	}
 
+	if ic.ControlPlane != nil && ic.ControlPlane.Platform.Ovirt != nil {
+		allErrs = append(
+			allErrs,
+			validateMachinePool(
+				con,
+				field.NewPath("controlPlane", "platform", "ovirt"),
+				ic.ControlPlane.Platform.Ovirt)...)
+	}
+	for idx, compute := range ic.Compute {
+		fldPath := field.NewPath("compute").Index(idx)
+		if compute.Platform.Ovirt != nil {
+			allErrs = append(
+				allErrs,
+				validateMachinePool(
+					con,
+					fldPath.Child("platform", "ovirt"),
+					compute.Platform.Ovirt)...)
+		}
+	}
+
 	return allErrs.ToAggregate()
 }
 
+func validateMachinePool(con *ovirtsdk.Connection, child *field.Path, pool *ovirt.MachinePool) field.ErrorList {
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, validateInstanceTypeID(con, child, pool))
+	return allErrs
+}
+
+func validateInstanceTypeID(con *ovirtsdk.Connection, child *field.Path, machinePool *ovirt.MachinePool) *field.Error {
+	if machinePool.InstanceTypeID != "" {
+		_, err := con.SystemService().
+			InstanceTypesService().InstanceTypeService(machinePool.InstanceTypeID).Get().Send()
```

Comment:
I'm not big fan of mixed indentation here, one multi-line the rest is one-line. single line is better even for this imo.

Reply:
for me shorter lines (< 120) are preferred for the price of a bit of weirdness of go multi-line. Unfortunately ovirt sdk is more verbose than I'd wanted it to be (a lot of client code with builder probably)

Reply:
For me shorter lines (< 120) are preferred for the price of a bit of weirdness of go multi-line. Unfortunately ovirt sdk is quite verbose (a lot of client code are the same probably...)

If that's a code-base guideline that's fine by me. I don't want to repeat this over again.